frontend/src/app/services/user/user.service.ts

```diff
@@ -75,3 +75,9 @@ export class UserService {
     });
   }
 }
+
+export interface User {
+  id: number;
+  name: string;
+  role: 'user' | 'administrator';
+}
```

Comment:
It could be a nice occasion to switch the moved interfaces to types.

Reply:
Done.